hooks/src/index.js

```diff
@@ -259,20 +253,27 @@ function afterNextFrame(callback) {
 		setTimeout(callback);
 	};
 	const timeout = setTimeout(done, RAF_TIMEOUT);
-	const raf = requestAnimationFrame(done);
+
+	let raf;
+	if (typeof window !== 'undefined') {
+		raf = requestAnimationFrame(done);
+	}
 }
 
-/* istanbul ignore else */
-if (typeof window !== 'undefined') {
-	let prevRaf = options.requestAnimationFrame;
-	afterPaint = newQueueLength => {
-		if (newQueueLength === 1 || prevRaf !== options.requestAnimationFrame) {
-			prevRaf = options.requestAnimationFrame;
+// Note: if someone used options.debounceRendering = requestAnimationFrame,
+// then effects will ALWAYS run on the NEXT frame instead of the current one, incurring a ~16ms delay.
+// Perhaps this is not such a big deal.
```

Comment:
pinging @robertknight because afaik this hook is used in `enzyme-adapter-preact-pure`.

Reply:
The Enzyme adapter uses the `debounceRendering` hook in order to be able to synchronously flush pending state updates at certain points. I can't see an issue with the change here.